src/dash/controllers/RepresentationController.js

```diff
@@ -266,7 +267,7 @@ function RepresentationController() {
         };
 
         updating = false;
-        eventBus.trigger(Events.AST_IN_FUTURE, { delay: delay });
+        eventBus.trigger(MediaPlayerEvents.AST_IN_FUTURE, { delay: delay });
```

Comment:
Please change back to Event ref as all events are aggregated into Event.js at runtime for easy access within Dash.js.